unlock_indicator.c

```diff
@@ -205,21 +202,19 @@ static cairo_font_face_t *font_faces[5] = {
  * Pro 13" Retina screen, the scaling factor is 227/96 = 2.36.
  *
  */
-static double scaling_factor(void) {
-    const int dpi = (double) screen->height_in_pixels * 25.4 /
-                    (double) screen->height_in_millimeters;
-    return (dpi / 96.0);
+static double calculate_scaling_factor(void) {
+    const double dpi = (double)screen->height_in_pixels * 25.4 /
```

Comment:
This needs to be changed back to an `int`.

I did some testing and with this as a double, my laptop ends up with a scaling factor of `1.00000985` instead of just `1.000...`. Looks like we don't want the extra precision here, I guess.